us.zoom.Zoom.json

```diff
@@ -10,16 +10,21 @@
     ],
     "finish-args": [
         "--share=ipc",
+        "--socket=wayland",
         "--socket=x11",
         "--socket=pulseaudio",
         "--share=network",
         "--device=all",
         "--filesystem=~/Documents/Zoom:create",
         "--filesystem=~/.zoom:create",
-        "--env=QT_QPA_PLATFORM=",
+        "--filesystem=/tmp/zoom:create",
```

Comment:
Here you add this, and then immediately removes it in the next commit. I'd rather not add it here, and remove the next commit.

Reply:
Yeah... I was playing around with a bunch of different configurations initially. Ideally we can just squash my commits if this ever gets merged?